.taskcluster.yml

```diff
@@ -206,6 +210,34 @@ tasks:
           owner: mcastelluccio@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - $if: 'tasks_for == "github-pull-request" && "Train with task cluster:" in pr_description'
+        then:
+          dependencies:
+            - { $eval: as_slugid("docker_build") }
+          taskId: { $eval: as_slugid("train_on_taskcluster") }
+          created: { $fromNow: "" }
+          deadline: { $fromNow: "1 day" }
+          provisionerId: proj-bugbug
+          workerType: compute-large
+          payload:
+            maxRunTime: 10800
+            image:
+              type: task-image
+              path: public/bugbug/bugbug-base.tar.zst
+              taskId: { $eval: as_slugid("docker_build") }
+            env:
+              TEXT_TO_EXTRACT: "${pr_description}"
+            command:
+              - "/bin/sh"
+              - "-lcx"
+              - 'model=$(python -c ''import re, os; match = re.search(r"Train with task cluster:(.+)", os.environ.get("TEXT_TO_EXTRACT")); model = match.group(1).strip() ; print(model)'') &&
```

Comment:
Why don't we do this logic inside `scripts.trainer`?

Reply:
I was trying to maintain the conditions of the arguments required to train the model. i.e. `model` is a required argument when running the training script. Is it okay to change this behaviour? since at the moment of running the script, with an option like `--auto-detect-model-name`, the model_name is not provided.

Reply:
I see. We could create a script in the `scripts` directory and use it to get the model name. This will be more flexible, testable, and maintainable.

Reply:
Please check the new script and let me know what you think.

Reply:
Thank you for your work. Sorry that my comment may not be clear. I did not mean to create a new trnner script. I meant to convert the code that was in the `python -c` to a standalone code and then use it to pass the options to the model trainer.

Example:
```
python -m scripts.trainer $(python -m scripts.trainer_extract_args)
```